android/app/src/main/java/org/fossasia/openevent/activities/SpeakerDetailsActivity.java

```diff
@@ -319,6 +320,16 @@ private static int getDarkColor(int color) {
     public boolean onOptionsItemSelected(MenuItem item) {
         switch (item.getItemId()) {
             case R.id.share_speakers_url:
+                StringBuilder sessionsBuilder = new StringBuilder();
+                boolean firstSession = true;
+                for (Session session : sessions) {
+                    if (!firstSession) {
+                        sessionsBuilder.append(", ");
+                    }
+                    sessionsBuilder.append(session.getTitle() + session.getShortAbstract());
+                    firstSession = false;
+                }
+
```

Comment:
Move this to a static utility method to share this

Reply:
Done. Please review